### PR TITLE
Fix/ListItem link - DAO-609

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- makes icon clickable
- fix lint warning and error

### CODE REVIEW

**Review by RakeshUP — APPROVED** [review]

File: packages/ui-components/src/components/listItem/link.tsx [hunk 1]
```diff
@@ -21,17 +21,16 @@ export const ListItemLink: React.FC<ListItemLinkProps> = ({
...
-        <IconLinkExternal className="w-1.5 h-1.5" />
-      </TitleContainer>
+      <Link
+        rel="noopener noreferrer"
+        {...props}
+        {...(external ? {target: '_blank'} : {})}
+        data-testid="listItem-link"
+      >
+        <Title>{props.label ? props.label : props.href}</Title>
+        <Icon />
```
Comment (WissenIstNacht) [inline packages/ui-components/src/components/listItem/link.tsx:31]: Shouldn't this be dependenent on the `external` prop?

**Review by WissenIstNacht — APPROVED** [review]
Thanks :)

File: packages/ui-components/src/components/listItem/link.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/ui-components/src/components/listItem/link.tsx:31]: Do you mean the Icon? I think that is built in.

Comment (WissenIstNacht) [reply packages/ui-components/src/components/listItem/link.tsx:31]: I see. But then the external prop seems redundant. Don't worry about it for now, just was just curious :)

Comment (Fabricevladimir) [reply packages/ui-components/src/components/listItem/link.tsx:31]: I'll revisit it during refactor. The external is for how the link is opened, whether in the same tab or in a new one. I wanted to set new tab by default, that's why I added it.

**Pull request merged**